docker/fullnode/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+---
+version: "3.9"
+services:
+  fullnode:
+    image: mysten/sui-node:devnet-0.4.0
+    ports:
```

Comment:
need to add here the 'expose' ports section for those 2 ports.

Reply:
done, thanks